src/util/image/MapProperties.java

```diff
@@ -78,13 +78,11 @@ public MapProperties() {
     COLOR_MAP.put("Impassible", new Color(0xD8BA7C));
   }
 
-  @SuppressWarnings("rawtypes")
-  public Tuple<PropertiesUI, List<MapPropertyWrapper>> propertyWrapperUI(final boolean editable) {
+  public Tuple<PropertiesUI, List<MapPropertyWrapper<Object>>> propertyWrapperUI(final boolean editable) {
```

Comment:
This needs a TODO comment at least -  'Object' is equivalent to rawTypes. Look to the usages and see if there is casting, a TODO comment that indicates the set of possible types would be of benefit. The ideal fix would be create an interface which each client could use instead.